### PR TITLE
Fix/rtenu 200 display current location

### PR DESCRIPTION
**Pull request by noravux**

### CODE REVIEW

File: packages/frontend/src/components/NavBar/MenuList.tsx [hunk 1]
```diff
@@ -18,30 +20,84 @@ export const MenuList = () => {
...
                     key={child.key}
                     href={child.to ? child.to : ''}
+                    selected={child.to === pathname}
                   >
-                    <ListItemText primary={child.primary} />
+                    <ListItemText
+                      primary={child.primary}
+                      sx={{
+                        '.MuiListItemText-primary': {
+                          fontWeight: selected ? 'bold' : 'normal',
```
Comment (NinaDang97) [inline packages/frontend/src/components/NavBar/MenuList.tsx:97]: Default font-family "Exo2-Regular" doesn't work with font-weight (I don't know why). To set it bold, you could instead use `fontFamily: 'Exo2-Bold' or 'Exo2-ExtraBold'`

File: packages/frontend/src/components/NavBar/MenuList.tsx [hunk 2]
```diff
@@ -18,30 +20,84 @@ export const MenuList = () => {
...
+                }}
+              >
                 <ListItemIcon>{icon}</ListItemIcon>
-                <ListItemText primary={primary} sx={{ opacity: openDrawer ? 1 : 0 }} />
+                <ListItemText
+                  primary={primary}
+                  sx={{
+                    opacity: openDrawer ? 1 : 0,
+                    '.MuiListItemText-primary': {
+                      fontWeight: selected ? 'bold' : 'normal',
```
Comment (NinaDang97) [inline packages/frontend/src/components/NavBar/MenuList.tsx:64]: Same as commented below, you could try using `fontFamily: 'Exo2-Bold' or 'Exo2-ExtraBold'` for bold text

**Review by NinaDang97 — CHANGES_REQUESTED** [review]

**Comment by NinaDang97** [comment]
Otherwise LGTM 👍, I'll approve after this fix

**Review by saneDG — APPROVED** [review]

File: packages/frontend/src/components/NavBar/MenuList.tsx (hunk 1 above)
Comment (noravux) [reply packages/frontend/src/components/NavBar/MenuList.tsx:97]: Weird, it works for me 🤔 If I set it to either of those two, then it won't work. What browser are you using where the bold font doesn't work with just the `bold` setting?

Comment (NinaDang97) [reply packages/frontend/src/components/NavBar/MenuList.tsx:97]: I'm using chrome 112.0.5615.137

**Review by NinaDang97 — APPROVED** [review]

File: packages/frontend/src/components/NavBar/MenuList.tsx (hunk 1 above)
Comment (noravux) [reply packages/frontend/src/components/NavBar/MenuList.tsx]: Ok it works now. But yeah, weird that the `fontWeight: bold` works as it should on Firefox but not on Chrome. I used the fontFamily option to go around this for now 👍

**Pull request merged**